frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -328,6 +319,29 @@ const Map: React.FC<MapProps> = ({
             />
             {selectedProperty && renderPopup(selectedProperty)}
             <LegendControl />
+            <LayersControl position="topright">
+              <LayersControl.Overlay checked={showParcelBoundaries} name="Parcel Boundaries">
```

Comment:
How are you controlling what layers are being displayed by default?

Reply:
The `checked` property of the Overlay is set to `true` to show the layer by default